x-pack/plugins/infra/server/lib/adapters/nodes/lib/create_node_item.ts

```diff
@@ -57,8 +58,11 @@ export function createNodeItem(
   node: InfraBucket,
   bucket: InfraBucket
 ): InfraNode {
+  const nodeDoc = first(get(node, 'nodeDetails.hits.hits'));
```

Comment:
This would probably be equivalent to

```suggestion
  const nodeDoc = get(node, ['nodeDetails', 'hits', 'hits', 0]);
```

without the need for `first()`.